docs/connectors/postgresql.asciidoc

```diff
@@ -1314,6 +1341,13 @@ In such cases it is possible to switch to so-called *streaming* mode when every
 |`false`
 |Whether or not to drop the logical replication slot when the connector finishes orderly. Should only be set to `true` in testing or development environments. Dropping the slot allows WAL segments to be discarded by the database, so it may happen that after a restart the connector cannot resume from the WAL position where it left off before.
 
+|`publication.name`
+|`dbz_publication`
+|The name of the PostgreSQL publication created created for streaming changes when using `pgoutput`.
+
+This publication is created at start-up if it does not already exist to include _all tables_.  Debezium will then use its own white-/black- list filtering capabilities to limit change events to the specific tables of interest if configured.
```

Comment:
Ah, that's interesting. I thought we'd rely this to be always created manually upfront?

Reply:
I believe we likely did make that final call and I had completely forgotten.  Do we want to make that adjust for the next 0.10 release or leave it as is @gunnarmorling ?